tests/phpunit/tests/comment.php

```diff
@@ -1672,4 +1672,225 @@ public function test_unspam_should_invalidate_comment_cache() {
 
 		$this->assertSame( '1', $comment->comment_approved );
 	}
+
+	/**
+	 * Tests that trashing a top-level note also trashes all direct child notes.
+	 *
+	 * @covers ::wp_trash_comment
+	 * @covers ::wp_trash_comment_children
```

Comment:
```suggestion
	 * @covers ::wp_trash_comment
```

The `wp_trash_comment_children()` function is no longer there.

Reply:
The tests are also missing the Trac ticket number.

Reply:
I will create a matching Trac ticket.